dracut/30ignition/ignition-generator

```diff
@@ -4,6 +4,11 @@
 
 set -e
 
+IGNITION_CONF="/etc/ignition.conf"
```

Comment:
I'm not sure we want to make this a singular config file like this. In general with things like dracut and whatnot, configuration supported as drop-in snippets has been better and easier for supporting a mixture of admin and vendor supplied configuration.